actor.go

```diff
@@ -1,23 +1,20 @@
 package troupe
 
 import (
-	"errors"
-	"sync"
+	"fmt"
+	"sync/atomic"
 	"time"
 )
 
 // Actor is an actor, who receives messages and acts over them
 type Actor struct {
-	errorHandler  func(error)
-	agent         chan Work
-	quit          chan struct{}
-	shutdownMutex sync.RWMutex
-	shutdown      bool
-	acceptMutex   sync.RWMutex
-	lastAccepted  time.Time
-	busyMutex     sync.RWMutex
-	busy          bool
-	lastFinished  time.Time
+	errorHandler func(error)
+	agent        chan Work
+	quit         chan struct{}
+	lastAccepted *int64
+	busy         *int32
+	lastFinished *int64
+	ID           int
```

Comment:
Ah, that's leftover garbage.

I was doing some internal distribution testing of Assign VS AssignRand, and removed it before pushing up the latest changes. I was giving each Actor a unique ID, and forgot to remove the ID field.